minitest/views.py

```
@@ -0,0 +1,229 @@
+                "data": [_user_to_dict(user) for user in users]
+            })
+        except APIException as e:
+            return Response({
+                "message": "error",
+                "data": str(e)
+            })
+        
+    def put(self, request):
+        try:
+            data = json.loads(request.body)
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)